packages/google_atlas/lib/src/google_atlas.dart

```diff
@@ -56,14 +57,27 @@ class _GoogleMapsProviderState extends State<GoogleMapsProvider> {
   Widget build(BuildContext context) {
     _onPositionUpdate();
 
-    return GoogleMaps.GoogleMap(
-      myLocationEnabled: showMyLocation,
-      myLocationButtonEnabled: showMyLocationButton,
-      mapType: GoogleMaps.MapType.normal,
-      initialCameraPosition: _toGoogleCameraPosition(position),
-      markers: markers.map((m) => _toGoogleMarker(m)).toSet(),
-      onTap: _toGoogleOnTap(onTap),
-      onMapCreated: _onMapCreated,
+    return FutureBuilder(
+      future: _toGoogleMarkers(markers),
+      builder: (BuildContext context,
+          AsyncSnapshot<Set<GoogleMaps.Marker>> snapshot) {
+        if (snapshot.data == null) {
+          return Container(
+            height: 0,
```

Comment:
I think there's no need to specify `height: 0, width: 0`, since those will be the defaults, and can save you a few lines of code... however, you might want to give it a `Key` and test the case where `snapshot.data` is null bi finding this widget by Key.